style.css

```diff
@@ -385,6 +388,61 @@ textarea:-ms-input-placeholder {
   margin-bottom: 8px;
 }
 
+/* MOBILE MENU */
+
+.mobile-menu-style {
+  background-color: #6070ff;
+  mix-blend-mode: multiply;
+  backdrop-filter: blur(5px);
+  background-size: cover;
+  height: 100vh;
+  width: 100%;
+  position: fixed;
+}
```

Comment:
- [ ] [OPTIONAL]
Nice job implementing recommended changes, however, the changes appear to affect our design. The blurr background effect is less better than it used to. Let's ensure that the blur effect matches the same for the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=0-1&t=uYcHolPGApJuKWca-0).

- [ ] [OPTIONAL]
Let's also ensure that the _hamburger_ and the _close_ button (X) are in the same position when toggled, so as to enhance the UX. 

**Your design:**        |  **The original design:**
:-------------------------:|:-------------------------:
<img width="800" alt="Home Page" src="https://github.com/HFG43/Portfolio_Mobile_First/assets/12745474/960f4941-88ad-4c41-8f5d-9514c731b828"> | <img width="800" alt="Home Page" src="https://github.com/HFG43/Portfolio_Mobile_First/assets/12745474/7b232719-21eb-464f-8ef9-ee9d4147a196">